windows-agent/internal/proservices/landscape/executor_test.go

```diff
@@ -205,7 +198,13 @@ func TestInstall(t *testing.T) {
 
 			if tc.appxDoesNotExist || tc.sendRootfsURL != "" {
 				// WSLMock Install only accepts ubuntu-22.04
-				settings.name = wsltestutils.RandomDistroName(t)
+				if tc.distroName == nil {
+					settings.name = wsltestutils.RandomDistroName(t)
```

Comment:
just checking, but no chance that the random name will match a reserved one?

Reply:
That's the pattern returned by `wsltestutils.RandomDistroName()`:
`return fmt.Sprintf("%s_%s_%d", testDistroPrefix, name, rand.Uint64())`
So, no way to match our reserved names unless we change that drastically.
I'm adding a comment about that just in case that test helper changes in the future, we're advised. I don't think that duplicating the reserved names check is worth the trouble.

Reply:
Agreed!